boards/BMS/BMS.ioc

```diff
@@ -13,17 +13,18 @@ ADC1.EnableInjectedConversion=ENABLE
 ADC1.EnableRegularConversion=ENABLE
 ADC1.ExternalTrigConv=ADC_EXTERNALTRIGCONV_T3_TRGO
 ADC1.ExternalTrigConvEdge=ADC_EXTERNALTRIGCONVEDGE_RISING
-ADC1.IPParameters=ClockPrescaler,Resolution,DataAlign,ScanConvMode,ContinuousConvMode,DiscontinuousConvMode,DMAContinuousRequests,EOCSelection,Overrun,LowPowerAutoWait,EnableRegularConversion,NbrOfConversion,ExternalTrigConv,ExternalTrigConvEdge,EnableInjectedConversion,InjNumberOfConversion,EnableAnalogWatchDog1,EnableAnalogWatchDog2,EnableAnalogWatchDog3,Rank-0\#ChannelRegularConversion,Channel-0\#ChannelRegularConversion,SamplingTime-0\#ChannelRegularConversion,OffsetNumber-0\#ChannelRegularConversion,Offset-0\#ChannelRegularConversion,master,SubFamily
+ADC1.IPParameters=ClockPrescaler,Resolution,DataAlign,ScanConvMode,ContinuousConvMode,DiscontinuousConvMode,DMAContinuousRequests,EOCSelection,Overrun,LowPowerAutoWait,EnableRegularConversion,NbrOfConversion,ExternalTrigConv,ExternalTrigConvEdge,EnableInjectedConversion,InjNumberOfConversion,EnableAnalogWatchDog1,EnableAnalogWatchDog2,EnableAnalogWatchDog3,Rank-0\#ChannelRegularConversion,Channel-0\#ChannelRegularConversion,SamplingTime-0\#ChannelRegularConversion,OffsetNumber-0\#ChannelRegularConversion,Offset-0\#ChannelRegularConversion,master,SingleDiff-0\#ChannelRegularConversion,SubFamily
 ADC1.InjNumberOfConversion=0
 ADC1.LowPowerAutoWait=DISABLE
 ADC1.NbrOfConversion=1
-ADC1.Offset-0\#ChannelRegularConversion=0
-ADC1.OffsetNumber-0\#ChannelRegularConversion=ADC_OFFSET_NONE
+ADC1.Offset-0\#ChannelRegularConversion=1983
```

Comment:
if possible give this constant a descriptive name on cube as well, maybe something like `TS_VOLTAGE_OFFSET`

Reply:
hm ic, lets see toms if there is a way to do this